chart/kubeapps/Chart.yaml

```diff
@@ -25,4 +25,4 @@ maintainers:
 name: kubeapps
 sources:
   - https://github.com/kubeapps/kubeapps
-version: 7.0.0
+version: 7.0.1-dev1
```

Comment:
@antgamdia I bumped the chart here, so it's a pre-release of 7.0.1 (ie, < 7.0.1 according to semver), but the important thing is that this shouldn't trigger a PR to the upstream chart either right? (as in, the comparison for creating the upstream chart  would need to compare with the current release + 0.0.1 (ie. check if 7.0.1-dev1 > 7.0.1, which it's not... rather than checking if 7.0.1-dev1 > 7.0.0, which it is). Does that make sense? I'll just revert the chart bump here if you don't have time to deal with that for now.

Reply:
> this shouldn't trigger a PR to the upstream chart either right?
No, our local changes will NOT trigger any update in the bitnami/charts repo, unless we are releasing (i.e., creating a tag)

> (as in, the comparison for creating the upstream chart would need to compare with the current release + 0.0.1 (ie. check if 7.0.1-dev1 > 7.0.1, which it's not... rather than checking if 7.0.1-dev1 > 7.0.0, which it is). 

What we are doing on every push a master is to check:

```bash 
# before bitnami accepts the PR, the remote version will be 7.0.0
semverCompare=$(semver compare "7.0.1-dev1" "7.0.0") # 1
  # Do nothing, our chart is more updated than the remote one, that is OK
...

# after bitnami accepts the PR, the remote version will be 7.0.1
semverCompare=$(semver compare "7.0.1-dev1" "7.0.1") # -1
# The current version is less than the chart external version, then retrieve the changes and send an internal PR with them
# But, given that some changes are already there, we will have a 7.0.1 version with almost no changes.
# In fact, if it is the same set of changes, we only get a chart version update.
```

> Does that make sense? 

Yes, and no. I mean: it is a bit odd sending duplicated PRs in both repos. If we happen to add any other change, we will have a, let's say, version 7.0.1 which does not match the upstream one. In that case, we would have to manually modify the PR and set it to 7.0.2-dev.

Reply:
Perfect. For some reason I thought we were sending a PR the other way if we intentionally made the chart version greater... but what you've described is much saner: we tag a release to trigger a PR to bitnami/charts, while we pull any new chart changes down. Great, thanks.